code/messaging/src/main/java/com/adobe/marketing/mobile/messaging/EdgePersonalizationResponseHandler.java

```diff
@@ -363,13 +373,13 @@ private void dispatchNotificationEventForSurfaces(final List<Surface> requestedS
     }
 
     /**
-     * Retrieves the previously fetched (and cached) feeds content from the SDK for the provided
+     * Dispatches an event with previously cached content cards from the SDK for the provided
      * surfaces.
      *
      * @param surfaces A {@code List<Surface>} of surfaces to use for retrieving cached content
      * @param event The retrieve message {@link Event}
      */
-    void retrieveMessages(final List<Surface> surfaces, final Event event) {
+    void retrieveCachedContentCards(final List<Surface> surfaces, final Event event) {
```

Comment:
This method is called inside the `serialWorkDispatcher`. We should make sure all global maps and lists (example: `contentCardsBySurface`) used are thread safe. We can take it it up in a separate PR.